ocaml/idl/datamodel.ml

```diff
@@ -1515,7 +1515,9 @@ let _ =
   error Api_errors.pusb_vdi_conflict [ "PUSB"; "VDI" ]
     ~doc:"The VDI corresponding to this PUSB has existing VBDs." ();
   error Api_errors.vm_has_vusbs ["VM"]
-    ~doc:"The operation is not allowed when the VM has VUSBs." ()
+    ~doc:"The operation is not allowed when the VM has VUSBs." ();
+  error Api_errors.pusb_not_in_possible_hosts [ "pusb"; "possible_hosts" ]
+    ~doc:"The host of the pusb which will be attached to the vm must in possible hosts that the vm can boot on." ()
```

Comment:
I think the error message is not clear and would suggest this one unless it does not capture the problem:
```
The VM can't boot on the host that belongs to the pusb; it can boot only on the following hosts.
```

Reply:
Better error message. Thanks